guides/v2.3/performance-best-practices/advanced-js-bundling.md

```diff
@@ -20,7 +20,7 @@ In a modular application, the number of server requests can reach into the hundr
 ![noBundling](images/noBundling.png)
 
 ### Magento merging and bundling
-Out of the box, Magento provides two ways to reduce the number of server requests: merging and bundling. These settings are turned off by default. You can turn them on within Magento's Admin UI in Stores > Configuration > Advanced > Developer > JavaScript Settings, or from the command line.
+Out of the box, Magento provides two ways to reduce the number of server requests: merging and bundling. These settings are turned off by default. You can turn them on within Magento's Admin UI in **Stores** > **Configuration** > **ADVANCED** > **Developerv** > **JavaScript Settings**, or from the command line.
```

Comment:
"Advanced" doesn't need to be capitalized, and you've added an extra "v" to "Developer".